web-crawler/src/main/java/com/google/sps/webcrawler/ContentProcessor.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.webcrawler;
+
+import com.google.sps.data.NewsArticle;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+/** 
+ * Provides a tool for processing textual content.
```

Comment:
Got it! I'll specify the processing methods. Right now it's just taking the first 100 words, and later I'll implement summarization. There may be [multiple processing methods](https://docs.google.com/document/d/1J8AfCorIeCkzOax94ubKtfowB2rbVNPsEGP1-vyhKwM/edit#bookmark=id.km2ddfszn8z2) in the future.

Reply:
Ok, I'll make changes to `NewsArticle`. Thanks! :)

Reply:
I created a new instance variable `abbreviatedContent` of type String to store the first `MAX_WORD_COUNT` words. The variable `content` would not get modified.

I also modified the database schema to store `content` as a list of text strings. Since the database doesn't allow indexed text strings to exceed 1500 bytes, I excluded these text strings from indexing.

Reply:
On second thought, I believe using a list of content blocks doesn't add much functionality to the web app. I will still format `content` as a string in the database, but excluded from indexing.

Reply:
In that case, would it make more sense to have `content` be a single string here as well?

Reply:
Sure, I'll make the changes

Reply:
This change makes the code significantly more succinct.